WBooks/Models/Book.swift

```diff
@@ -15,5 +18,16 @@ struct Book {
     let id: Int
     let author: String
     let title: String
-    let imageUrl: String
+    let imageUrl: String?
```

Comment:
You could make this directly a `URL?` type, since you don't want the String, you want the URL.
You can make this by making URL Decodable if it is not, or by using custom function to decode:
```
    <*> json <|? "image_url" >>- toURL


private func toURL(urlString: String) -> Decoded<URL> {
    return Decoded<URL>.fromOptional(URL(string: urlString))
}
```